service_capacity_modeling/models/org/netflix/graphkv.py

```diff
@@ -0,0 +1,206 @@
+                    )
+                else:
+                    item_size_gib = 1 / 1024**2
+                item_count = user_desires.data_shape.estimated_state_size_gib.scale(
+                    1 / item_size_gib
+                )
+            # assume 512 B to track the id and metadata write_ts of each item
+            relaxed.data_shape.estimated_state_size_gib = item_count.scale(
+                512 / 1024**3
+            )
+            return relaxed
+
+        return (("org.netflix.key-value", _modify_kv_desires),)
+
+    @staticmethod
+    def default_desires(user_desires, extra_model_arguments):
+        if user_desires.query_pattern.access_pattern == AccessPattern.latency:
+            return CapacityDesires(
+                query_pattern=QueryPattern(
+                    access_pattern=AccessPattern.latency,
+                    access_consistency=GlobalConsistency(
+                        same_region=Consistency(
+                            target_consistency=AccessConsistency.read_your_writes,
+                        ),
+                        cross_region=Consistency(
+                            target_consistency=AccessConsistency.eventual,
+                        ),
+                    ),
+                    estimated_mean_read_size_bytes=Interval(
+                        low=128, mid=1024, high=65536, confidence=0.95
+                    ),
+                    estimated_mean_write_size_bytes=Interval(
+                        low=64, mid=128, high=1024, confidence=0.95
+                    ),
+                    estimated_mean_read_latency_ms=Interval(
+                        low=0.2, mid=1, high=2, confidence=0.98
+                    ),
+                    estimated_mean_write_latency_ms=Interval(
+                        low=0.2, mid=1, high=2, confidence=0.98
+                    ),
+                    # "Single digit milliseconds SLO"
+                    read_latency_slo_ms=FixedInterval(
+                        minimum_value=0.2,
+                        maximum_value=10,
+                        low=1,
+                        mid=3,
+                        high=6,
+                        confidence=0.98,
+                    ),
+                    write_latency_slo_ms=FixedInterval(
+                        minimum_value=0.2,
+                        maximum_value=10,
+                        low=0.4,
+                        mid=2,
+                        high=5,
+                        confidence=0.98,
+                    ),
+                ),
+                data_shape=DataShape(
+                    estimated_state_size_gib=Interval(
+                        low=10, mid=50, high=200, confidence=0.98
+                    ),
+                    reserved_instance_app_mem_gib=8,
+                ),
+            )
+        else:
+            return CapacityDesires(
+                query_pattern=QueryPattern(
+                    access_pattern=AccessPattern.latency,
```

Comment:
is this the right access_pattern? its hard to tell what is the difference between these two branches